Makefile

```diff
@@ -1,36 +1,45 @@
 # Builds PEP files to HTML using sphinx
 
 PYTHON=python3
-VENV_DIR=venv
+VENVDIR=.venv
 JOBS=8
-RENDER_COMMAND=$(PYTHON) build.py -j $(JOBS)
+RENDER_COMMAND=$(VENVDIR)/bin/python3 build.py -j $(JOBS)
```

Comment:
If one installs `make` on windows, this would fail, right? As the executables directory is `Scripts`.

A

Reply:
I don't use `make` on Windows, I've often seen a separate `make.bat` though so I think that is used instead of `Makefile`.

* https://github.com/python/cpython/blob/main/Doc/Makefile
* https://github.com/python/cpython/blob/main/Doc/make.bat

Reply:
@AA-Turner correct, this is Unix-only. The `PYTHON` variable could be updated to use `VENVDIR`, but that would also allow for overriding the path to the Python executable.

Reply:
I do have `make` on my Windows dev machines and use it when necessary sometimes, but at least on this repo I don't really do so, and instead run the various commands I need directly (usually with custom invocations). I wouldn't expect very many if any non-WSL Windows users to do so either.